src/components/CreateList.tsx

```diff
@@ -29,7 +29,10 @@ export function CreateList({ user, setListPath }: Props) {
 			toast.success("Success: Your New List is Created!");
 			// Delay for toast notification before redirecting
 			setTimeout(() => {
-				navigate("/list");
+				console.log(path);
```

Comment:
whose that pokemon?!

it's a...

console.log!!

😂😅

Reply:
LOL! I guess I didn't catch em all !!!